source/operations/troubleshooting.rst

```diff
@@ -185,6 +185,71 @@ You can disable upload at any time using :mc-cmd:`mc support callhome disable`.
 Use :mc:`mc admin logs` command to display logs from the command line.
 The command supports type and quantity filters for further limiting logs output.
 
+Upgrades and Version Support
+----------------------------
+
+MinIO releases versions of its software as often as needed to introduce features, improve performance, address security concerns, or fix bugs. 
+These releases can occur very frequently, and vary by product.
+
+Always test upgrades in a development environment before upgrading on a production deployment.
```

Comment:
Q: Do we want to use "Upgrades" or "Updates" ? 

I have always thought of "Upgrade" as the verb and "Update" as the thing, but I guess they are interchangeabl?

Reply:
But here, "Test upgrades before upgrading" -> "Test updates before upgrading" or "Test new software releases...before upgrading"

Reply:
Going with "Always test software releases ..."